.github/workflows/test.yaml

```diff
@@ -0,0 +1,74 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - '**/*'
+
+jobs:
+  test-android-sdk:
+    runs-on: macos-latest
+    env:
+      ANDROID_API_LEVEL: 33
```

Comment:
I learned that this env is not necessary

Reply:
Ok will axe! 🪓